ui/src/app/views/app/component.ts

```diff
@@ -44,5 +52,19 @@ export class AppComponent implements OnInit {
         }
       }
     });
+    this.user = this.signInService.user;
+
+    this.user.subscribe(u => {
+      console.log('USER', u);
```

Comment:
OK to trim out this logging? I think the usage in the template will make the code run?